client/apps/settings/stylesheets/index.styl

```diff
@@ -28,3 +28,49 @@
     color gray-darker-color
   &:last-of-type
     border-bottom none
+
+.page-header.sticky
```

Comment:
This should probably live [here](https://github.com/artsy/positron/blob/74c5611d004687347efa0109e1582b9db94d0fb5/client/components/layout/stylesheets/index.styl#L28) to co-locate the styling with the original `.page-header` definition. It's minor, but for example—leaving in local to the app could make it easy to for someone to forget to port over this generic style when faced with rebuilding and app (something not often in Positron, but we do a lot in Force).